blazor/dashboard-layout/getting-started.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Getting Started with Blazor Dashboard Layout Component | Syncfusion
-description: Checkout and learn about getting started with Blazor Dashboard Layout component in Blazor Server App and Blazor WebAssembly App.
+description: Learn here all about getting started with Blazor Dashboard Layout component in Blazor Server App and Blazor WebAssembly App.
```

Comment:
Revert this and remove unwanted space only

Reply:
Reverted